proto/cosmos/crypto/secp256r1/keys.proto

```diff
@@ -0,0 +1,22 @@
+syntax = "proto3";
+package cosmos.crypto.secp256r1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/crypto/keys/secp256r1";
+option (gogoproto.messagename_all) = true;
+option (gogoproto.goproto_stringer_all) = false;
+option (gogoproto.goproto_getters_all) = false;
+
+// PubKey defines a secp256r1 ECDSA public key.
+message PubKey {
+  // Point on secp256r1 curve in a compressed representation as specified in section
+  // 4.3.6 of ANSI X9.62.
+  bytes key = 1 [(gogoproto.customtype) = "ecdsaPK"];
+}
+
+// PrivKey defines a secp256r1 ECDSA private key.
+message PrivKey {
+  // secret number serialized using big-endian encoding
+  bytes secret = 1 [(gogoproto.customtype) = "ecdsaSK"];
```

Comment:
note: we use `key` in ed25519 and secp256k1. but I like secret, so ok to leave it like this.

Reply:
Yes. It's not exposed by the interface, and it better says what it does -> a single secret number instead of `PrivKey,Key` (name clash).